src/harness/testcases/testdata/sas_impl_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "sas_impl/0/0",
+  "name": "Sas Implementation Record",
+  "administratorId": "sas_admin/sas/0",
+  "contactInformation": {
+    "contactType": "operation",
+    "name": "Contact Name",
+    "phoneNumber": ["123 456 7890"],
+    "email": ["admin@sas.com"],
+    "address": ["Address"],
+    "note": ["Test Data for Harness"]
+  },
```

Comment:
comment on "sas_impl_record_0.json" object using by "WINNF_FT_S_SIR_testcase.py" in master branch
ContactInformation should be a array 
Refer to the last WINNF version for SAS-SAS protocol
https://workspace.winnforum.org/higherlogic/ws/public/download/3403/WINNF-16-S-0096-V1.5.8%20SAS-SAS%20Protocol%20Technical%20Specification.pdf